rocketpy/Rocket.py

```diff
@@ -851,7 +856,7 @@ def info(self):
 
         # Show plots
         print("\nAerodynamics Plots")
-        self.powerOnDrag()
+        self.plots.powerOnDrag()
 
```

Comment:
I think we cante this out of the `info` method. We decided to only leave prints in the `info`. All plots go to allInfo